Vixen.System/Sys/PropertyManager.cs

```diff
@@ -35,6 +35,21 @@ public IPropertyModuleInstance Add(IPropertyModuleInstance instance)
 			return instance;
 		}
 
+		public IPropertyModuleInstance AddNoDefaulting(IPropertyModuleInstance instance)

```

Comment:
Was this the intended method name? Seems odd or I am not getting the meaning.

Reply:
Maybe AddWithoutDefaults might be a little more descriptive. After looking broader at the other code, I see this is a clone of the Add method without setting the default values. So maybe that would help others make that connection faster than I did.